src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -78,22 +82,25 @@ public void storeMessage(Message message) {
   /**
    * Get List of messages posted by a specific user.
    *
-   * @return a list of messages posted by the user, or empty list if user has never posted a
+   * @return a list of messages posted by the user, or empty list if user has
+   *     never posted a
```

Comment:
formatting here is a bit weird i think?

Reply:
My formatter borked, I've updated it. (It was breaking the comment at 80 chars)